### ISSUE TITLE
Can't get axes to be fixed in 3D scatter plot

### ISSUE THREAD

**Issue by meder411** [facebookresearch/visdom#451]
I'm trying to visualize pointclouds in the 3D scatter plot. Everything plots fine, except my axes keep changing with each update. The annoying consequence is that the visualization is distorted (as the axes are not all scaled similarly), making it quite difficult to tell if anything is working. It seems from the [examples](https://github.com/facebookresearch/visdom/blob/master/example/demo.py#L102) that `xtickmax` and the like should do the trick, but it doesn't seem to be. What am I missing?

I am using the most recent visdom available via `pip`.

My code snippet:

```
vis.scatter(
	X=pts,
	Y=labels,
	win='points',
	opts=dict(
		title='Points',
		legend=['Predicted', 'Ground Truth'],
		markersize=3,
		markercolor=torch.tensor([[0,0,255], [255,0,0]]).int().numpy(),
		xtickmin=-10,
		xtickmax=10,
		xtickstep=0.5,
		ytickmin=-10,
		ytickmax=10,
		ytickstep=0.5,
		ztickmin=-10,
		ztickmax=10,
		ztickstep=0.5))
```

Output snapshots:

![image](https://user-images.githubusercontent.com/6818607/44326818-bbfb2d80-a411-11e8-9ffd-1a5945686a2a.png)
![image](https://user-images.githubusercontent.com/6818607/44327004-48a5eb80-a412-11e8-9199-fc2e679c3188.png)

**Comment by JackUrb** [comment]
Are you providing the axis parameters to the opts dict on every update step? Do they work the first time and just not on subsequent updates?

**Comment by meder411** [comment]
Every update. They actually never seem to have any affect.

**Comment by JackUrb** [comment]
It's possible that something isn't working properly with 3dplots and plotly - I can investigate sometime this week.

**Comment by meder411** [comment]
Any update on this? Also, could [this](https://github.com/facebookresearch/visdom/issues/449) be related?

**Comment by JackUrb** [comment]
Hi @meder411, sorry for the delay here - I haven't ended up getting much visdom bandwidth in the last two weeks but I'm going to be tackling the backlog this Wednesday.

I don't think that issue is directly related, as I believe heatmaps require particular formatting on the plotly end that visdom isn't configured to handle yet. It could be similar as far as visdom just not being configured yet, as I'm not sure ztick settings have been implemented at all yet.

**Comment by JackUrb** [comment]
Hi @meder411 - Try installing from source, this should work for you now. As an aside, `tickstep` is ignored on 3d plots
